services/server/config/config.go

```diff
@@ -116,6 +118,7 @@ func (c *Config) ValidateV2() error {
 // GRPCConfig provides GRPC configuration for the socket
 type GRPCConfig struct {
 	Address        string `toml:"address"`
+	TTRPCAddress   string `toml:"ttrpc_address"`
```

Comment:
This is a mistake right? You just need the `TTRPC` section above?

Reply:
Shoot, I thought I removed this. Yes it's a mistake.